Kudu.Core/Helpers/EnvironmentHelper.cs

```diff
@@ -31,11 +32,17 @@ public static bool IsWindowsContainers()
             return isolation == "hyperv" || isolation == "process";
         }
 
+        public static bool IsRunFromPackage()
+        {
+            string runFromPackage = System.Environment.GetEnvironmentVariable("WEBSITE_RUN_FROM_PACKAGE");
+            return !string.IsNullOrEmpty(runFromPackage) && !StringUtils.IsFalseLike(runFromPackage);
```

Comment:
also OR with Uri.TryParse - it could be URI format as well.

Reply:
Sounds good. Right now, it checks if `WEBSITE_RUN_FROM_PACKAGE` exists and is not set to `0` or `false`, then we assume it's set to a correct value.
But I can add an explicit URL check as you suggest.